extensions/rich_text_components/Image/directives/oppia-noninteractive-image.directive.ts

```diff
@@ -91,7 +91,6 @@ angular.module('oppia').directive('oppiaNoninteractiveImage', [
               ctrl.filepath);
           } catch (e) {
             var additionalInfo = (
-              '\nExploration ID: ' + ContextService.getExplorationId() +
```

Comment:
Why removing this?

Reply:
Since this function (getExplorationId) throws an error if not called in an exploration editor (as there is no explorationId there).
Also, in cases it is in an exploration editor/viewer, `entityId` would be `explorationId`  and `entityType` would be `exploration`. So, there is no loss of information in removing this.